src/test/kotlin/org/rust/lang/core/completion/RsCompletionTest.kt

```diff
@@ -775,6 +775,19 @@ class RsCompletionTest : RsCompletionTestBase() {
         use bar::foo;/*caret*/
     """)
 
+    @ProjectDescriptor(WithDependencyRustProjectDescriptor::class)
+    fun `test complete bang proc macro unqualified`() = doSingleCompletionByFileTree("""
```

Comment:
Let's also add tests for attribute macro completion (and for caret position after insertion)